docs/dev/table/connectors/kafka.zh.md

```diff
@@ -0,0 +1,213 @@
+
+<span class="label label-danger">Attention</span> The Kafka SQL sink does not support exactly-once writes to Kafka.
+
+The dependency downloading links is only available for stable releases.
+
+Flink's streaming connectors are not currently part of the binary distribution.
+See how to link with them for cluster execution [here]({{ site.baseurl}}/dev/projectsetup/dependencies.html).
+
+How to create a Kafka table
+----------------
+<div class="codetabs" markdown="1">
+<div data-lang="SQL" markdown="1">
+{% highlight sql %}
+CREATE TABLE kafkaTable (
+ user_id BIGINT,
+ item_id BIGINT,
+ category_id BIGINT,
+ behavior STRING,
+ ts TIMESTAMP(3)
+) WITH (
+ 'connector' = 'kafka',
+ 'topic' = 'user_behavior',
+ 'properties.bootstrap.server' = 'localhost:9092',
+ 'properties.group.id' = 'testGroup',
+ 'format' = 'csv',
+ 'scan.startup.mode' = 'earliest-offset'
+)
+{% endhighlight %}
+</div>
+</div>
+
+Connector Options
+----------------
+
+<table class="table table-bordered">
+    <thead>
+    <tr>
+      <th class="text-left" style="width: 25%">Option</th>
+      <th class="text-center" style="width: 8%">Required</th>
+      <th class="text-center" style="width: 7%">Default</th>
+      <th class="text-center" style="width: 10%">Type</th>
+      <th class="text-center" style="width: 50%">Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td><h5>connector</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Specify what connector to use, for Kafka the options are: 'kafka', 'kafka-0.11', 'kafka-0.10'.</td>
+    </tr>
+    <tr>
+      <td><h5>topic</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Topic name from which the table is read.</td>
+    </tr>
+    <tr>
+      <td><h5>properties.bootstrap.servers</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Kafka server connection string.</td>
+    </tr>
+    <tr>
+      <td><h5>properties.group.id</h5></td>
+      <td>required by source</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Consumer group in Kafka consumer, no need for Kafka producer</td>
+    </tr>
+    <tr>
+      <td><h5>format</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Kafka connector requires to specify a format,
+      the supported formats are 'csv', 'json' and 'avro'.
+      Please refer to [Table Formats]({{ site.baseurl }}/dev/table/connect.html#table-formats) section for more details.
```

Comment:
This link is not rendered: 

![image](https://user-images.githubusercontent.com/5378924/84356810-4f10e300-abf7-11ea-852a-0f7dc1c42103.png)